### PR TITLE
feature/implement-header

### PR DESCRIPTION
**Pull request by phu-tran-thien**

### CODE REVIEW

File: src/lib/Header.svelte [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+    margin-right: 35px;
+  }
+
+  .header__top__right__social:after {
+    position: absolute;
+    right: -20px;
+    top: 1px;
+    height: 20px;
+    width: 1px;
+    background: #000000;
```
Comment (vanrin12) [inline src/lib/Header.svelte:152]: Should define variable for color and reuse

File: src/lib/Header.svelte [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+    opacity: 0.1;
+    content: '';
+  }
+
+  .header__top__right__social a {
+    font-size: 14px;
+    display: inline-block;
+    color: #1c1c1c;
```
Comment (vanrin12) [inline src/lib/Header.svelte:160]: Should  define variable

File: src/lib/Header.svelte (hunk 1 above)
Comment (phu-tran-thien) [reply src/lib/Header.svelte]: Hi a @vanrin12 , I have fixed it. Thank you.

File: src/lib/Header.svelte (hunk 2 above)
Comment (phu-tran-thien) [reply src/lib/Header.svelte]: Hi a @vanrin12 , I have fixed it. Thank you.

**Review by vanrin12 — APPROVED** [review]

**Pull request merged**